build.gradle

```diff
@@ -68,3 +68,19 @@ idea {
         sourceDirs += file('src/api/java')
     }
 }
+
+test {
+    workingDir = {minecraft.runDir + "/test"} // This can be set to whatever you prefer
```

Comment:
Can't this be made the default dir in barometer, as this will almost always be just this value.

Reply:
The `workingDir` property is a part of Gradle's testing job, not Barometer. If not set, it defaults to the project directory, which causes a bit of a mess when Barometer starts the test server in that place.

I had originally intended for Barometer to only depend on JUnit and not have any direct interaction with Gradle, but I now realize that I already depend on ForgeGradle already, so perhaps having some cleaner integration with Gradle would not be unnecessary bloat.

Figuring out how to properly integrate with Gradle may take some time as I have no experience with it, but I'll do what I can.